### ISSUE TITLE
GEPA reflection minibatch size is akin to learning_rate. We should have a LR schedule strategy support that can modify it dynamically

### ISSUE THREAD

**Issue by LakshyAAAgrawal** [gepa-ai/gepa#132]
The reflection minibatch size for training is sort of like a learning rate step size? If I give the optimizer 15 negative examples to learn from it should update the prompt a lot more than if I only give it 3. There could be support for some sort of schedule?
